### PR TITLE
issue comment

### PR DESCRIPTION
**Pull request by abhisheks-prophecy**
### Fix Made

Explain the fix that was applied to achieve the expected behaviour

### Asana Ticket

Enter the Asana ticket link against which fix was made (comma separated if more than 1)

Asana Ticket Links: << https://app.asana.com/0/<project_id>/<issue_id1>,https://app.asana.com/0/<project_id>/<issue_id2> >>

### CODE REVIEW

**Comment by abhisheks-prophecy** [comment]
hello

File: .github/workflows/add_tag.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ name: Pull Request
 on:
   pull_request:
     types: [opened, synchronize, closed]
+  issue_comment:
```
Comment (abhisheks-prophecy) [inline .github/workflows/add_tag.yml:6]: new comment

**Comment by abhisheks-prophecy** [comment]
hello

**Comment by abhisheks-prophecy** [comment]
hi

**Comment by abhisheks-prophecy** [comment]
test

**Comment by abhisheks-prophecy** [comment]
dasdsa

File: .github/workflows/add_tag.yml [hunk 2]
```diff
@@ -3,6 +3,8 @@ name: Pull Request
 on:
   pull_request:
     types: [opened, synchronize, closed]
+  pull_request_review_comment:
```
Comment (abhisheks-prophecy) [inline .github/workflows/add_tag.yml:6]: new

**Comment by abhisheksr8** [comment]
test

File: .github/workflows/add_tag.yml (hunk 2 above)
Comment (abhisheksr8) [inline .github/workflows/add_tag.yml:6]: rest

**Pull request merged**